tensorflow/python/ops/image_ops_impl.py

```diff
@@ -464,6 +464,12 @@ def flip_left_right(image):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+  >>> image = np.array([[[1], [2]], [[3], [4]]])
```

Comment:
Please add an empty line after `Usage Example:`. Also, `Example` should be `example`.